ShiftPlan.Blazor.Client.Tests/LocalFileServiceTests.cs

```diff
@@ -0,0 +1,43 @@
+using FluentAssertions;
+using ShiftPlan.Blazor.Client.Services;
+using ShiftPlan.Blazor.Commons.Models;
+using ShiftPlan.Blazor.Commons.Services;
+
+namespace ShiftPlan.Blazor.Client.Tests
+{
+    public class LocalFileServiceTests
+    {
+        private string jsonFile = Path.Combine( Directory.GetParent(AppDomain.CurrentDomain.BaseDirectory).FullName,"Employee.json");
```

Comment:
I think Employee.json should be in folder, for example `TestData`
```suggestion
        private string jsonFilePath = Path.Combine("./TestData/Employee.json");
```